File: Migration_Project/Backend/ChienVHShopOnline/Models/Order.cs
namespace ChienVHShopOnline.Models
{
    public class Order
    {
        public Order()
        {
            OrderDetails = new HashSet<OrderDetail>();
        }

        public int Id { get; set; }
        public string OrderName { get; set; }
        public DateTime? OrderDate { get; set; }
        public string PaymentType { get; set; }
        public string Status { get; set; }
        public string CustomerName { get; set; }
        public string CustomerPhone { get; set; }
        public string CustomerEmail { get; set; }
        public string CustomerAddress { get; set; }
        public int UserId { get; set; } 
        public virtual User User { get; set; }

        public virtual ICollection<OrderDetail> OrderDetails { get; set; }
    }
}

File: Migration_Project/Frontend/ChienVHShopOnlineUI/src/app/models/order.ts
import { OrderDetailModel } from "./order-detail";

export interface OrderModel {
  id: number;
  orderName: string;
  orderDate: string;
  paymentType: string;
  status: string;
  customerName: string;
  customerPhone: string;
  customerEmail: string;
  customerAddress: string;
  orderDetails: OrderDetailModel[];
}